go/host/rpc/enclaverpc/enclave_client.go

```diff
@@ -6,6 +6,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/obscuronet/go-obscuro/go/common/retry"
```

Comment:
I thought the linter picked up on these 🤔 
Can you arrange the imports ?

Reply:
Forgot to lint